pkg/occlient/occlient_test.go

```diff
@@ -406,3 +408,883 @@ func Test_parseImageName(t *testing.T) {
 		})
 	}
 }
+
+func TestClient_UpdateDCAnnotations(t *testing.T) {
+	tests := []struct {
+		name        string
+		dcName      string
+		annotations map[string]string
+		existingDc  appsv1.DeploymentConfig
+		wantErr     bool
+	}{
+		{
+			name:   "dc name with annotations",
+			dcName: "nodejs",
+			annotations: map[string]string{
+				"app.kubernetes.io/url":                   "file:///temp/nodejs-ex",
+				"app.kubernetes.io/component-source-type": "local",
+			},
+			existingDc: appsv1.DeploymentConfig{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "nodejs",
+					Annotations: map[string]string{"app.kubernetes.io/url": "https://github.com/sclorg/nodejs-ex",
+						"app.kubernetes.io/component-source-type": "git",
+					},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name:   "dc name with annotations",
```

Comment:
Can you give it a different name from the first one. WIll cause confusion while running tests.